kind/kind.mk

```diff
@@ -1,13 +1,9 @@
 kind_dir ?= $(PWD)/.kind
-kind_bin = $(kind_dir)/kind
+kind_bin = $(GOPATH)/bin/kind
 
 # Prepare kind binary
-# We need to set the Go arch since the binary is meant for the user's OS.
-$(kind_bin): export GOOS = $(shell go env GOOS)
-$(kind_bin): export GOARCH = $(shell go env GOARCH)
 $(kind_bin):
-	@mkdir -p $(kind_dir)
-	cd kind && go build -o $@ sigs.k8s.io/kind
+	go install sigs.k8s.io/kind@latest
```

Comment:
This isn't working for me... for some reason I have to execute the `go install` outside the make:
```
gmake local-install
go install sigs.k8s.io/kind@latest
/Users/simonbeck/go/bin/kind create cluster \
        --name provider-cloudscale-v1.24.0 \
        --image docker.io/kindest/node:v1.24.0 \
        --config kind/config.yaml
/bin/bash: /Users/simonbeck/go/bin/kind: No such file or directory
gmake: *** [kind/kind.mk:36: /Users/simonbeck/repos/appcat/appcat-service-s3/.kind/kind-kubeconfig-v1.24.0] Error 127

go install sigs.k8s.io/kind@latest

gmake local-install               
/Users/simonbeck/go/bin/kind create cluster \
        --name provider-cloudscale-v1.24.0 \
        --image docker.io/kindest/node:v1.24.0 \
        --config kind/config.yaml
Creating cluster "provider-cloudscale-v1.24.0" ...
 ✓ Ensuring node image (docker.io/kindest/node:v1.24.0) 🖼 
 ✓ Preparing nodes 📦  
 ✓ Writing configuration 📜 
```

Reply:
Strange, I did not spot any make target dependency issues.
Can you reproduce it with debug flag? (`make -d`) so I can better understand what's happening.

Reply:
```
gmake -d local-install
GNU Make 4.3
Built for x86_64-apple-darwin21.1.0
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Reading makefiles...
Reading makefile 'Makefile'...
Reading makefile 'Makefile.vars.mk' (search path) (no ~ expansion)...
Reading makefile 'docs/antora-preview.mk' (search path) (don't care) (no ~ expansion)...
Reading makefile 'docs/antora-build.mk' (search path) (don't care) (no ~ expansion)...
Reading makefile 'kind/kind.mk' (search path) (don't care) (no ~ expansion)...
Reading makefile 'test/local.mk' (search path) (don't care) (no ~ expansion)...
Reading makefile 'package/package.mk' (search path) (don't care) (no ~ expansion)...
Updating makefiles....
 Considering target file 'package/package.mk'.
  Looking for an implicit rule for 'package/package.mk'.
  No implicit rule found for 'package/package.mk'.
  Finished prerequisites of target file 'package/package.mk'.
 No need to remake target 'package/package.mk'.
 Considering target file 'test/local.mk'.
  Looking for an implicit rule for 'test/local.mk'.
  No implicit rule found for 'test/local.mk'.
  Finished prerequisites of target file 'test/local.mk'.
 No need to remake target 'test/local.mk'.
 Considering target file 'kind/kind.mk'.
  Looking for an implicit rule for 'kind/kind.mk'.
  No implicit rule found for 'kind/kind.mk'.
  Finished prerequisites of target file 'kind/kind.mk'.
 No need to remake target 'kind/kind.mk'.
 Considering target file 'docs/antora-build.mk'.
  Looking for an implicit rule for 'docs/antora-build.mk'.
  No implicit rule found for 'docs/antora-build.mk'.
  Finished prerequisites of target file 'docs/antora-build.mk'.
 No need to remake target 'docs/antora-build.mk'.
 Considering target file 'docs/antora-preview.mk'.
  Looking for an implicit rule for 'docs/antora-preview.mk'.
  No implicit rule found for 'docs/antora-preview.mk'.
  Finished prerequisites of target file 'docs/antora-preview.mk'.
 No need to remake target 'docs/antora-preview.mk'.
 Considering target file 'Makefile.vars.mk'.
  Looking for an implicit rule for 'Makefile.vars.mk'.
  No implicit rule found for 'Makefile.vars.mk'.
  Finished prerequisites of target file 'Makefile.vars.mk'.
 No need to remake target 'Makefile.vars.mk'.
 Considering target file 'Makefile'.
  Looking for an implicit rule for 'Makefile'.
  No implicit rule found for 'Makefile'.
  Finished prerequisites of target file 'Makefile'.
 No need to remake target 'Makefile'.
Updating goal targets....
Considering target file 'local-install'.
 File 'local-install' does not exist.
  Considering target file 'kind-load-image'.
   File 'kind-load-image' does not exist.
    Considering target file 'kind-setup'.
     File 'kind-setup' does not exist.
     Considering target file '/Users/simonbeck/repos/appcat/appcat-service-s3/.kind/kind-kubeconfig-v1.24.0'.
      File '/Users/simonbeck/repos/appcat/appcat-service-s3/.kind/kind-kubeconfig-v1.24.0' does not exist.
      Considering target file '/Users/simonbeck/go/bin/kind'.
       File '/Users/simonbeck/go/bin/kind' does not exist.
       Finished prerequisites of target file '/Users/simonbeck/go/bin/kind'.
      Must remake target '/Users/simonbeck/go/bin/kind'.
go install sigs.k8s.io/kind@latest
Putting child 0x7fe2a8f08f80 (/Users/simonbeck/go/bin/kind) PID 19734 on the chain.
Live child 0x7fe2a8f08f80 (/Users/simonbeck/go/bin/kind) PID 19734 
Reaping winning child 0x7fe2a8f08f80 PID 19734 
Removing child 0x7fe2a8f08f80 PID 19734 from chain.
      Successfully remade target file '/Users/simonbeck/go/bin/kind'.
      Finished prerequisites of target file '/Users/simonbeck/repos/appcat/appcat-service-s3/.kind/kind-kubeconfig-v1.24.0'.
     Must remake target '/Users/simonbeck/repos/appcat/appcat-service-s3/.kind/kind-kubeconfig-v1.24.0'.
/Users/simonbeck/go/bin/kind create cluster \
        --name provider-cloudscale-v1.24.0 \
        --image docker.io/kindest/node:v1.24.0 \
        --config kind/config.yaml
Putting child 0x7fe2a900c100 (/Users/simonbeck/repos/appcat/appcat-service-s3/.kind/kind-kubeconfig-v1.24.0) PID 19749 on the chain.
Live child 0x7fe2a900c100 (/Users/simonbeck/repos/appcat/appcat-service-s3/.kind/kind-kubeconfig-v1.24.0) PID 19749 
/bin/bash: /Users/simonbeck/go/bin/kind: No such file or directory
Reaping losing child 0x7fe2a900c100 PID 19749 
gmake: *** [kind/kind.mk:36: /Users/simonbeck/repos/appcat/appcat-service-s3/.kind/kind-kubeconfig-v1.24.0] Error 127
Removing child 0x7fe2a900c100 PID 19749 from chain.
```

Reply:
But maybe don't invest too much time here. Rather implement the `$GOBIN` as discussed below. Maybe that already fixes the issues.

Reply:
I've done the GOBIN part. Please try again.